compiler/rustc_hir_analysis/src/errors.rs

```diff
@@ -320,7 +320,6 @@ pub struct AutoDerefReachedRecursionLimit<'a> {
 #[diag(hir_analysis_track_caller_on_main)]
 pub(crate) struct TrackCallerOnMain {
     #[primary_span]
-    pub span: Span,
     #[label]
-    pub annotated: Span,
```

Comment:
```
#[primary_span]
#[suggestion(applicability = "maybe-incorrect", code = "")]
pub span: Span,
#[label]
pub annotated: Span,
```

Then make sure the ftl file has a `.suggestion = ...`

Reply:
I made a new commit, thanks.

What does `"maybe-incorrent"` and `"machine-applicable"` mean?

Reply:
They correspond to the enum variants in https://doc.rust-lang.org/nightly/nightly-rustc/rustc_errors/enum.Applicability.html